phac-nml/irida-next
# Fix create project form errors

## What does this PR do and why?
Updated Project Form to display errors when Project is not created due to failing validation, refer to Group Form on how we detect and display errors.
Fixes #171.

## Screenshots or screen recordings
![Screenshot from 2023-09-15 09-52-32](https://github.com/phac-nml/irida-next/assets/325703/e568b4c5-5657-4f2b-89a3-57948b8276c1)
![Screenshot from 2023-09-15 09-55-08](https://github.com/phac-nml/irida-next/assets/325703/8eaf12df-ebdd-4c71-8d12-2155478af259)
![Screenshot from 2023-09-15 12-42-30](https://github.com/phac-nml/irida-next/assets/325703/90770b8e-1360-4b34-bff5-e35ce9603e7d)

## How to set up and validate locally
1. From the Projects Dashboard click `New project`
2. Fill in `Project Name` and `Path` with `a`
3. Click `Create project`
4. See that the form is re-rendered and fields show errors are displayed.
5. Navigate back to Projects Dashboard and confirm that no project is created as expected.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
